cmd/ssl-handshake-check/main.go

```diff
@@ -36,26 +71,92 @@ func init() {
 		log.Error("ERROR: The PORT environment variable has not been set.")
 		return
 	}
+	selfSigned = os.Getenv("SELF_SIGNED")
+	if len(selfSigned) == 0 {
+		log.Error("ERROR: The SELF_SIGNED environment variable has not been set.")
+		return
+	}
+	selfSignedBool, _ = strconv.ParseBool(selfSigned)
+
+	// set debug mode for nodeCheck pkg
+	nodeCheck.EnableDebugOutput()
 }
 
 func main() {
-	err := runHandshake()
+	// create context
+	checkTimeLimit := time.Minute * 1
+	ctx, _ := context.WithTimeout(context.Background(), checkTimeLimit)
```

Comment:
same comment here as in https://github.com/Comcast/kuberhealthy/pull/658/files#r498511466

Reply:
Updated, PR resubmitted.